common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -16,49 +22,115 @@ interface PackageJson {
   devDependencies: Record<string, string>;
 }
 
+/**
+ * The default sourcemaps plugin does not provide very much information in warnings, so this shim allows us to capture
+ * the active sourcemaps loading context.
+ *
+ * This allows us to selectively disable warnings about missing source maps, for example in core-asynciterator-polyfill.
+ */
+export function sourcemapsExtra() {
+  const _sourcemaps = sourcemaps();
+
+  const load = _sourcemaps.load;
+
+  if (!load) return _sourcemaps;
+
+  return Object.assign(_sourcemaps, {
+    load(this: PluginContext, id: string) {
+      const shim = new Proxy(this, {
+        get(context, p, ...rest) {
+          if (p === "warn") {
+            const warn = context.warn;
+            return (warning: unknown) => {
+              const warningObject = (
+                typeof warning === "string" ? { message: warning } : warning
+              ) as RollupWarning;
+
+              warningObject.id = id;
+
+              warn(warningObject);
+            };
+          }
+          return Reflect.get(context, p, ...rest);
+        },
+      });
+
+      return load.call(shim, id);
+    },
+  });
+}
+
 // #region Warning Handler
 
 /**
- * A function that can determine whether a rollupwarning should be ignored. If
+ * A function that can determine whether a rollup warning should be ignored. If
  * the function returns `true`, then the warning will not be displayed.
  */
 export type WarningInhibitor = (warning: RollupWarning) => boolean;
 
-function ignoreNiseSinonEvalWarnings(warning: RollupWarning): boolean {
+function ignoreNiseSinonEval(warning: RollupWarning): boolean {
   return (
     warning.code === "EVAL" &&
-    (warning.id?.includes("node_modules/nise") || warning.id?.includes("node_modules/sinon")) ===
-      true
+    (warning.id?.includes(["node_modules", "nise"].join(path.sep)) ||
```

Comment:
Dependency on nise was dropped in the new recorder. Since the old recorder is no longer present in the repo, can we do without this?

Reply:
Thanks! I'll take a look at this and remove it if it isn't actively inhibiting any warnings.

Reply:
CC @HarshaNalluru

Reply:
Oh yes, great point @joheredi. 🔥

Should be able to remove now.